torch/serialization.py

```diff
@@ -199,10 +205,47 @@ def _meta_deserialize(obj, location):
         return torch.UntypedStorage(obj.nbytes(), device='meta')
 
 
+def validate_privateuse1_device(location, backend_name):
+    device = torch.device(location)
+    device_index = device.index if device.index else 0
+    if not hasattr(torch, backend_name):
+        raise RuntimeError(f'The {backend_name.upper()} device module is not registered. '
+                           'If you are running on a CPU-only machine, '
+                           'please use torch.load with map_location=torch.device(\'cpu\') '
+                           'to map your storages to the CPU.')
```

Comment:
Will this actually work, if I don't have the backend module? 🤔

Reply:
I think this check is necessary, although it hardly triggers.

If user rename privateuse1 as foo by `torch.utils.rename_privateuse1_backend('foo')`, 
and generate `storage.foo()` method by `torch.utils.generate_methods_for_privateuse1_backend(for_tensor=False, for_module=False, for_storage=True)`,
and device module is not registered by `torch._register_device_module('foo', DeviceModule)`.

Maybe throw this exception.